site.yml

```diff
@@ -137,6 +137,7 @@
   tags:
     - filebeat
     - windows
+    - windows_filebeat
```

Comment:
Are you using these tags for something? Otherwise they can be removed.

Reply:
no, I was just undoing your previous commit without understanding how this tags worked